regression/abstraction/abst-funcs.c

```diff
@@ -0,0 +1,714 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+// Some helper functions.
+
+// nndt_bool, nndt_int, nndt_sizet are available in CBMC.
+
+size_t nndt_int()
```

Comment:
`size_t` and `int` may be quite different.

Reply:
Changed.